source/MapPanel.cpp

```diff
@@ -1173,11 +1186,12 @@ void MapPanel::DrawTooltips()
 	// Create the tooltip text.
 	if(tooltip.empty())
 	{
-		pair<pair<int, int>, int> t = escortSystems.at(hoverSystem);
+		pair<pair<int, int>, map<const Planet *, unsigned>> t = escortSystems.at(hoverSystem);
```

Comment:
`escortSystems` only gets populated when the `Show escort systems on map` preference is set. Thus, the extra information about outfits is currently conditional on that preference, too. I'm not sure what the best way to fix this is.

Reply:
Is rename to `Show escorts and stored outfits on map` an option?

Reply:
Combining escorts and stored outfits into a single setting sounds good; however:

- That string would overflow the space allotted to it in the preferences screen.

- If we rename the option, we'll need to teach the code to read the old name from existing preferences.txt files for backward compatibility. We might even continue to _write_ it, too, if we want to support downgrading to older versions of endless-sky without losing a change to the setting. (In theory, we could make this option the only one whose UI name is different to its preferences.txt name, but I don't think we want to do that.)

- In the longer term, introducing an abstraction between the UI name and the preferences.txt name of a setting wouldn't be the worst idea.

Reply:
So, the preferences screen looks like this:

![2021-05-19-002708_261x249_scrot](https://user-images.githubusercontent.com/24784687/118739422-f7342080-b838-11eb-8bfc-fa1272f22bab.png)

The option names don't currently have tooltips.

The option names must fit on a single line. (They're drawn with Table::Draw(std::string&), which renders the string on a single line.)

This being the case, I think it'll be easiest to add a separate `Show stored outfits on map` preference. Renaming the preference, aside from the backward compatibility preferences.txt reading consideration, would require teaching `Table` to have multiline cells. That seems like it would be a lot of effort for little gain, or possibly even a lot of effort for _negative_ gain, should support for multiline cells make it more difficult to extend Table in other ways in the future.

Reply:
> This being the case, I think it'll be easiest to add a separate `Show stored outfits on map` preference.

Done and force pushed. Should hopefully be good to merge now? The orbits tooltips needn't block adding the systems tooltip, I think.